client/package.json

```diff
@@ -6,28 +6,33 @@
     "dev": "vite dev",
     "build": "vite build",
     "preview": "vite preview",
-    "test": "playwright test",
+    "test:recaptcha": "playwright test --config playwright.config.recaptcha.ts",
+    "test:procaptcha": "playwright test --config playwright.config.procaptcha.ts",
+    "test": "yarn test:recaptcha && yarn test:procaptcha",
     "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
     "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch"
   },
   "devDependencies": {
-    "@playwright/test": "^1.28.1",
-    "@sveltejs/adapter-auto": "^2.0.0",
-    "@sveltejs/adapter-node": "^1.2.3",
-    "@sveltejs/adapter-static": "^2.0.1",
-    "@sveltejs/kit": "^1.15.2",
-    "@tailwindcss/typography": "^0.5.9",
-    "autoprefixer": "^10.4.14",
-    "daisyui": "^2.51.4",
-    "postcss": "^8.4.21",
-    "svelte": "^3.54.0",
-    "svelte-check": "^3.0.1",
-    "svelte-markdown": "^0.2.3",
-    "svelte-meta-tags": "^2.7.0",
-    "tailwindcss": "^3.2.7",
-    "tslib": "^2.4.1",
-    "typescript": "^4.9.3",
-    "vite": "^4.2.3"
+    "@playwright/test": "^1.40.0",
```

Comment:
- We need the polkadot deps to be bumped to match the versions in our bundle
- We need the react deps because yarn doesn't install react and we need it for procaptcha
- The prettier deps are in their eslint but missing from the package.json. I think this is a mistake but I'll remove it and see what happens.